app/code/Magento/Wishlist/etc/webapi.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+<routes xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Webapi:etc/webapi.xsd">
+    <route method="GET" url="/V1/wishlist/:customerId">
+        <service class="Magento\Wishlist\Api\WishlistRepositoryInterface" method="getWishlistForCustomer"/>
+        <resources>
+            <resource ref="anonymous"/>
```

Comment:
Please, use separate resources for customer and admin with corresponded access rights. 
Anonymous shouldn't have access to wishlist at all (the only customers may have wishlist)

Reply:
Who should have access? And how do I know who implements the rights? I'm looking at other files for examples but it is confusing. For instance, webapi.xml for lots of modules just use 'Magento_Catalog::products' but I can't find where this is defined. Do you have place with example I can look at?